src/ui/Analytics/LiveAnalyticsClient.ts

```diff
@@ -242,7 +242,7 @@ export class LiveAnalyticsClient implements IAnalyticsClient {
       splitTestRunName: this.splitTestRunName || result.splitTestRun,
       splitTestRunVersion: this.splitTestRunVersion || (result.splitTestRun != undefined ? result.pipeline : undefined),
       documentUri: result.uri,
-      documentUriHash: result.raw['urihash'],
+      documentUriHash: result.raw['uniqueid'] || result.raw['urihash'],
```

Comment:
`documentUriHash` should continue using `@urihash` as before. We only want to add the new info in the custom data when it's available.